### DISCUSSION TITLE
那个问一下我只想用这个工具测试IP延迟怎么搞呀

### DISCUSSION THREAD

**Discussion by woaiguyu1314** [XIU2/CloudflareSpeedTest#439] [提问]
那个问一下我只想用这个工具测试IP延迟怎么搞呀

**Comment by XIU2** [comment]
```
    -dd
        禁用下载测速；禁用后测速结果会按延迟排序 (默认按下载速度排序)；(默认 启用)
```

加上 `-dd` 这个运行参数后，就不会进行下载测速了，也就是只会单纯的延迟测速。

**Reply by woaiguyu1314** [reply]
好的谢谢，那个我想问一下为啥手机上运行cloudflarest只有延迟没有速度呀，同样代码同样IP库电脑上就有

**Reply by woaiguyu1314** [reply]
那个我测延迟显示可用ip好几百个我设置-p 9999 为啥还是不能输出来完呀，只输出十来个ip

**Reply by woaiguyu1314** [reply]
那个我测延迟显示可用ip好几百个我设置-p 9999 为啥还是不能输出来完呀，只输出十来个ip

**Comment by woaiguyu1314** [comment]
大佬那个我测延迟显示可用ip好几百个我设置-p 9999 为啥还是不能输出来完呀，只输出十来个ip

**Reply by XIU2** [reply]
提供个完整截图（包含你运行的 CloudflareST 命令参数 及 输出结果内容）。

**Reply by woaiguyu1314** [reply]
![17008845947846846351370601796662](https://github.com/XIU2/CloudflareSpeedTest/assets/140800587/13df6ea9-52d8-4a72-bead-4d3966b7b0ba)
就是类似这种

**Reply by XIU2** [reply]
因为你加了 `-tl 200` 参数，过滤掉 200ms 以上的，就剩下这 11 个 IP 了。

可用 IP 数量指的是延迟测速过程中，没有超时的 IP，和你设定的延迟条件无关，因为延迟测速后，还会根据延迟测速条件来过滤一遍，剩下的 IP 才会继续下载测速或像你这样直接输出结果（`-dd`）

**Reply by woaiguyu1314** [reply]
好的谢谢

**Reply by woaiguyu1314** [reply]
大佬问一下为啥我使用软件扫描ip段时，软件扫出来的结果是有延迟但是我ping这个ipping不通这是为啥呀
